bfh/__init__.py

```diff
@@ -209,7 +228,12 @@ def is_empty(self):
 
     @property
     def _raw(self):
-        return self
+        out = dict(**self.__dict__)
+        for name in self._field_names:
```

Comment:
GenericSchema's _field_names will always be empty. This is because it has no fields declared on it -- it's generic.

I'd expect some test to be failing right now. Your tests are pretty elaborate. Perhaps we also need some very basic tests of `_raw` on Schema and GenericSchema?

```
In [2]: wow = bfh.GenericSchema(foo="wow")

In [3]: wow.serialize()
Out[3]: {'foo': 'wow'}

In [4]: wow._field_names
Out[4]: []
```